bigsea-scaler/service/api/actuator/plugins/remote_KVM_tunnel.py

```diff
@@ -35,4 +54,18 @@ def get_allocated_resources(self, host_ip, vm_id):
         except:
             # FIXME: review this exception type
             raise Exception("Could not get allocated resources")
-        
\ No newline at end of file
+            
+    def get_io_quota(self, host_ip, vm_id):
+        command = "virsh blkdeviotune %s" % (vm_id)
+        command += " \"`virsh domblklist %s | awk 'FNR == 3 {print $1}'`\"" % (vm_id)
```

Comment:
It's a little bit confuse. I think it is better to change it to:
`command = "virsh blkdeviotune %s \"``virsh domblklist %s " + \n
                      " | awk 'FNR == 3 {print $1}'``\" " + \n
                      " | grep total_bytes_sec: | " + \n
                      "awk '{print $2}'" % (vm_id, vm_id)`